src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/SystemService.java

```diff
@@ -268,6 +304,44 @@ public void uploadAnchor(byte[] anchorBytes) throws InvalidAnchorInstanceExcepti
         }
     }
 
+    /**
+     * Is global conf initialized -> it is if whe can find a Configuration anchor
+     * @return
+     */
+    public boolean isAnchorImported() {
+        boolean isGlobalConfInitialized = false;
+        try {
+            AnchorFile anchorFile = getAnchorFile();
+            if (anchorFile != null) {
+                isGlobalConfInitialized = true;
+            }
+        } catch (AnchorNotFoundException e) {
+            log.info("SystemService#isAnchorImported: could not find Global Configuration Anchor", e);
```

Comment:
Hmm, is this a misleading log entry, will it look like an error for system admins, with the stacktrace? Could it maybe be reworded?

Reply:
Removed the logging since there is really nothing to log here. Just swallow the exception and continue